data-preprocessing/crop_riddle_script/crop_riddle_fs_video.py

```diff
@@ -0,0 +1,178 @@
+    """
+    Print usage instructions
+    """
+    print("Usage: python crop_riddle_fs.py [OPTIONS]")
+    print("")
+    print("Options:")
+    print("  -h, --help          Display this help message and exit")
+    print("  -v, --version       Display the script version and exit")
+    print("  -i, --instruction   Provide the path to the instruction file")
+    print("")
+    print("Usage Instructions:")
+    print("- Provide the root path of videos in the instruction file.")
+    print("- Provide the destination path of cropped videos in the instruction file.")
+    print("- If necessary, configure the target statements in the instruction file.")
+    print("- Ensure the videos in the root path end with an mp4 extension.")
+    sys.exit(0)
+
+# -------------------------------------------------
+# Function to read instruction file root directory
+#--------------------------------------------------
+def readInstructionFile():
+    """
+    Read instruction file
+    """
+    with open('ins.json', 'r') as ins:
+        data = json.load(ins)
+    return data
+
+# Function to validate json instruction file
+def check_json_file(filepath):
+    """
+    Check required keys inside json file
+    """
+    required_keys = ["root_path", "cropped_video_destination_path"]
+
+    with open(filepath) as f:
+        data = json.load(f)
+
+    for key in required_keys:
+        if key not in data:
+            raise ValueError("Check required parameters in instruction file")
+
+    return True
+
+# Check json file
+print("Checking instruction file...")
+check_json_file("ins.json")
+
+# Read instructions from file
+data = readInstructionFile() 
+    
+# Get root path and destination path from instructions
+root_path = data["root_path"] 
+
+# Check if root path is absolute
+print("Checking if root path provided is an absolute path...")
+if not os.path.isabs(root_path):
+   raise ValueError("Root path must be an absolute path")
+
+cropped_video_destination_path = data["cropped_video_destination_path"]
+
+# List to provide path(s) of videos to be transcribed
+video_paths = [os.path.join(root_path, f) for f in os.listdir(root_path)
+               if f.endswith('.mp4')]
+
+#------------------------------------------------------------------
+# Destination path
+# Create a new directory to store the cropped video and audio files 
+# specified in instruction file
+#--------------------------------------------------------------------
+if not os.path.exists(cropped_video_destination_path):
+    os.makedirs(cropped_video_destination_path)
+else:
+    # Seek permission to go ahead with existing folder
+    permission = input(f"Do you want to use the existing folder:{cropped_video_destination_path} for the cropped videos [y or n]: ")  # noqa: E501
+    if permission.lower() == "y":
+        print("Going ahead with the script...")
+    else:
+        raise ValueError("Please check destination folder of cropped videos in the instructions file")  # noqa: E501
```

Comment:
why do we need `noqa: E501`?

Reply:
It was turning off a prompt concerning line exceeding the line formatting convention.